src/test/ui/feature-gates/feature-gate-type_alias_enum_variants.rs

```diff
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
```

Comment:
Not sure why this copyright is still here... can you get rid of it?

------------

cc @Mark-Simulacrum 

Should we add a check to tidy for "Copyright 2018 The Rust Project Developers." perhaps ?

Reply:
A today check for copyright doesn't hurt; I haven't had the time to implement it myself unfortunately. I think someone was working on it but I forget who. PRs welcome, of course.

Reply:
Resolving this because of my PR.